sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AspectSentiment.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * The {@link AspectSentiment} model.
+ */
+@Immutable
+public final class AspectSentiment {
+    private final String text;
+    private final TextSentiment sentiment;
+    private final SentimentConfidenceScores confidenceScores;
+    private final int length;
+    private final int offset;
+
+    /**
+     * Create an {@link AspectSentiment} model that describes aspect.
+     *
+     * @param text The aspect text property.
+     * @param sentiment The text sentiment label: POSITIVE, NEGATIVE.
+     * @param offset The offset of aspect text.
+     * @param length The length of aspect text.
+     * @param confidenceScores The {@link SentimentConfidenceScores}.
+     */
+    public AspectSentiment(String text, TextSentiment sentiment,
+        int offset, int length, SentimentConfidenceScores confidenceScores) {
+        this.text = text;
+        this.sentiment = sentiment;
+        this.confidenceScores = confidenceScores;
+        this.length = length;
+        this.offset = offset;
+    }
+
+    /**
+     * Get the aspect text property.
+     *
+     * @return The text value.
+     */
+    public String getText() {
+        return text;
+    }
+
+    /**
+     * Get the aspect text sentiment label: POSITIVE, NEGATIVE.
```

Comment:
label?

Reply:
we are already using 'label' many places in the TA SDK.

Reply:
will delete label from all existing javadocs

Reply:
hmm. https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-sentiment-analysis?tabs=version-3
after review the service doc. they are also using label. So i think we should keep it.